wazuh/wazuh-splunk
# Fix server side table expand row

Hi team,

this fixes the expand row error feature in the server side table.

Closes #1111 

How to test:

1. Go to `Agents`
2. Select an Agent
3. Go to `Integrity Monitoring`
4. Click a row
5. See the row expand with additional information

![Peek 2021-09-07 17-29](https://user-images.githubusercontent.com/9343732/132371544-ec4d181e-91ba-4de0-bcfa-1a1f97c49312.gif)
